### PR TITLE
DEPLOY REVIEW

### PR DESCRIPTION
**Pull request by pmaxy**
### Things done

- Update  Readme files.
- Close check the project errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 Hello Microverse <a name="about-project"></a>
+# 📖 Portiflio Setup and mobile first <a name="" href="https://pmaxy.github.io/setup-and-mobile-first/">CHECK PAGE HERE</a>
```
Comment (emmiiorji) [inline README.md]: Following the [readme template](https://github.com/microverseinc/readme-template), kindly include the link within the live demo section of the readme file.

image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @pmaxy, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more

## Good Highlights✔️

- Project is live


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -17,6 +17,7 @@
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
```
Comment (TadesseDev) [inline README.md]: - [ ] Dear @pmaxy, I can see you have your link down the line but also as per the previous reviewer mentioned. For professional project documentation, we need to update the readme file to include the current live version of the project. Please add a link to the live version.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @pmaxy 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

199733072-bdacca19-8df9-43a7-be93-3aff2f1c15c8

 ## Highlights

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -53,6 +54,16 @@
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo" href="https://pmaxy.github.io/setup-and-mobile-first/" ></a>
+
+> Add a link to your deployed project.
+
+- [https://pmaxy.github.io/setup-and-mobile-first/]
```
Comment (emmiiorji) [inline README.md]: [Optional] In order to properly markup the link and not necessary show the full URL in text, you may use the following text for the line above:

Markup:
image

Display:
image

**Review by emmiiorji — APPROVED** [review]
Hi @Hassaanjbaig-code, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️

- Requested changes have been implemented.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (pmaxy) [reply README.md]: thank you alot i will make the changes

**Pull request merged**